dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/mapper/CommandMapperTest.java

```diff
@@ -179,6 +182,14 @@ public void testQueryCommandPageBySlot() {
         toTestQueryCommandPageBySlot(masterCount, thisMasterSlot);
     }
 
+    @Test
+    public void testClone() throws Exception {
+        Command command1 = createCommand();
+        Command command2 = createCommand();
+        Command cloneBean = (Command) BeanUtils.cloneBean(command1);
+        Assert.assertEquals(cloneBean.getId(), command1.getId());
+    }
```

Comment:
This is pointless, you are testing a third-party method, this is not our responsibility, instead, you should add a test case that verifies `createComplementDependentCommand` doesn't copy the id to the new dependent command

Reply:
Okay, I will modify it.